src/basic/glyph-util.c

```diff
@@ -111,6 +117,14 @@ const char *special_glyph_full(SpecialGlyph code, bool force_utf) {
                         [SPECIAL_GLYPH_ARROW_UP]                = u8"↑",       /* actually called: UPWARDS ARROW */
                         [SPECIAL_GLYPH_ARROW_DOWN]              = u8"↓",       /* actually called: DOWNWARDS ARROW */
 
+                        /* Progress bar */
+                        [SPECIAL_GLYPH_PROGRESS_FILL_START]     = u8"╺",       /* actually called: BOX DRAWINGS HEAVY RIGHT*/
+                        [SPECIAL_GLYPH_PROGRESS_FILL]           = u8"━",       /* actually called: BOX DRAWINGS HEAVY HORIZONTAL*/
+                        [SPECIAL_GLYPH_PROGRESS_FILL_END]       = u8"╸",       /* actually called: BOX DRAWINGS HEAVY LEFT*/
+                        [SPECIAL_GLYPH_PROGRESS_EMPTY_START]    = u8"╶",       /* actually called: BOX DRAWINGS LIGHT RIGHT*/
+                        [SPECIAL_GLYPH_PROGRESS_EMPTY]          = u8"─",       /* actually called: BOX DRAWINGS LIGHT HORIZONTAL*/
+                        [SPECIAL_GLYPH_PROGRESS_EMPTY_END]      = u8"╴",       /* actually called: BOX DRAWINGS LIGHT LEFT*/
```

Comment:
this should be redundant?

we have draw_progress_bar() after all now, which doesn't use this?